CorsixTH/Lua/humanoid_actions/use_staffroom.lua

```diff
@@ -114,7 +114,7 @@ local function use_staffroom_action_start(action, humanoid)
   local loop_callback_use = --[[persistable:use_staffroom_action_loop_callback]] function(obj_action)
     humanoid:wake(relaxation[action.target_type])
     -- if staff is no longer fatigued, make them leave the staff room
-    if humanoid.attributes["fatigue"] == 0 then
+    if humanoid.attributes.fatigue and humanoid.attributes.fatigue == 0 then
```

Comment:
Why change this line?

Reply:
To check fatigue exists before comparing it with 0.

Reply:
The result is the same, nil ~= 0

Reply:
Reverted